components/playlist/playlist_download_request_manager.cc

```diff
@@ -50,7 +62,11 @@ PlaylistDownloadRequestManager::PlaylistDownloadRequestManager(
       delegate_(delegate),
       media_detector_component_manager_(manager) {
   observed_.Observe(media_detector_component_manager_);
+
   media_detector_script_ = media_detector_component_manager_->script();
+
+  // TODO(sko) This line should be removed when component extension is ready.
```

Comment:
`component extension` is built-in extension (bundled by default). - https://source.chromium.org/chromium/chromium/src/+/main:extensions/docs/component_extensions.md

What we'll use here is `component`.

Reply:
Thanks for the correction! I thought there's something like 'built-in/bundled' extension haha